greenheart/tools/eco/electrolysis.py

```diff
@@ -99,20 +104,19 @@ def run_electrolyzer_physics(
             )
             * 1e-3,
         )
-        prodrate = 1.0 / 50.0  # kg/kWh
+        prodrate = 1.0 / round(H2_Results['new_H2_Results']['Rated BOL: Efficiency [kWh/kg]'],2) # kg/kWh
         roughest = energy_to_electrolyzer_kw * prodrate
         print("Energy to electrolyzer (kWh): ", sum(energy_to_electrolyzer_kw))
         print(
             "Energy per kg (kWh/kg): ",
-            energy_available * 1e3 / H2_Results["hydrogen_annual_output"],
+            H2_Results['new_H2_Results']['Sim: Total Input Power [kWh]'] / H2_Results['new_H2_Results']['Sim: Total H2 Produced [kg]'],
```

Comment:
because the old H2_Results is used everywhere so it'd be a headache to fix everyone's code and the new_H2_Results is much better (has units and more description keys). hydrogen_annual_output (I think) is the same as H2_Results['new_H2_Results']['Sim: Total H2 Produced [kg]'] - its redundant. But H2_Results['hydrogen_annual_output'] is littered throughout the code and the key isn't very descriptive (like - what are the units?)

I also kind of this think that this plotting stuff + print statements should be moved to a different function so that we don't have to pass in stuff (like wind_resource) to just run the electrolyzer.